Packs/AlienVault_OTX/Integrations/AlienVault_OTX_v2/AlienVault_OTX_v2.py

```diff
@@ -149,6 +147,28 @@ def create_pulse_by_ec(entry: dict) -> dict:
     return assign_params(**pulse_by_ec)
 
 
+def create_attack_pattern_relationships(client: Client, raw_response: dict, entity_a: str, entity_a_type: str):
+    relationships: list = []
+
+    if not client.create_relationships:
+        return relationships
+
+    # pulse_info.pulses.[0].attack_ids.display_name - can contain a list of attack_ids
+    pulses = dict_safe_get(raw_response, ['pulse_info', 'pulses'], [''])
+    if pulses and isinstance(pulses, list) and 'attack_ids' in pulses[0]:
+        display_names = [attack_id.get('display_name') for attack_id in pulses[0].get('attack_ids')]
+        if display_names:
+            relationships = [EntityRelation(
+                name=EntityRelation.Relations.INDICATOR_OF,
+                entity_a=entity_a,
+                entity_a_type=entity_a_type,
+                entity_b=display_name,
+                entity_b_type="Attack Pattern",
```

Comment:
I think that this is blocked by `Attack Pattern` indicator as this indicator dose not exist yet.
what we have at the moment is `STIX Attack Pattern`.

Reply:
correct. we map by the details fields which should indeed be `STIX Attack Pattern`.
https://github.com/demisto/content/blob/master/Packs/CommonTypes/IndicatorTypes/reputation-STIX_Attack_Pattern.json#L5

I can wither change to `STIX Attack Pattern` or wait. your call.

Reply:
yes, please change to `STIX Attack Pattern`